package.json

```diff
@@ -27,6 +27,7 @@
     "blanket": "1.1.6",
     "chai": "1.9.1",
     "grunt": "0.4.5",
+    "grunt-bump": "^0.6.0",
```

Comment:
Remove the `^` from the version number to ensure maximum compatibility in the future.